.github/workflows/main.yaml

```diff
@@ -21,7 +21,9 @@ concurrency:
 
 jobs:
   bazel:
-    runs-on: ubuntu-20.04
```

Comment:
What's the OS version of the custom runners?

Reply:
```bash
k exec dre-runner-custom-jlm9w-runner-65lbr -c runner -- cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

Reply:
To avoid glibc and other lib issues (again) we should use the oldest OS version possible. Can we use Ubuntu 20.04?

Reply:
Yeah, i will test it